web/app/mu-plugins/disallow-indexing.php

```diff
@@ -3,12 +3,20 @@
 Plugin Name:  Disallow Indexing
 Plugin URI:   https://roots.io/bedrock/
 Description:  Disallow indexing of your site on non-production environments.
-Version:      1.0.0
+Version:      2.0.0
 Author:       Roots
 Author URI:   https://roots.io/
+Text Domain:  roots
 License:      MIT License
 */
 
-if (defined('WP_ENV') && WP_ENV !== 'production' && !is_admin()) {
-    add_action('pre_option_blog_public', '__return_zero');
+if (!defined('WP_ENV') || WP_ENV === 'production') {
+    return;
 }
+
+add_action('pre_option_blog_public', '__return_zero');
```

Comment:
I'm pretty sure there needs to be `!is_admin()` check around this to prevent WordPress admin panel from always showing the option as enabled.